lib/readyset/configuration.rb

```diff
@@ -3,18 +3,26 @@
 
 module Readyset
   class Configuration
-    attr_accessor :enable_failover, :failover_error_window_period, :failover_error_window_size,
-      :failover_healthcheck_interval, :migration_path, :shard
+    attr_accessor :migration_path, :shard
 
     def initialize
-      @enable_failover = false
-      @failover_healthcheck_interval = 5.seconds
-      @failover_error_window_period = 1.minute
-      @failover_error_window_size = 10
       @migration_path = File.join(Rails.root, 'db/readyset_caches.rb')
```

Comment:
**:sunglasses::_** Moving the parameters to a `failover` method really represents the underlying behavior in a clean way. 

I could definitely learn from you in regards to how you're anticipating these sorts of caveats!

Reply:
It was your suggestion that prompted the refactor, so thank you 😄